test/solidity/oz/erc2771/forward.js

```diff
@@ -0,0 +1,188 @@
+/*-
```

Comment:
Are these files duplicated from erc2771 ticket?

Reply:
Kind of. Both tasks are using similar helper functions so I rebased this one from the Forwarder Task.
My plan was to merge the Forwarder first ... so the Comments and Diffs will disappear after we do so.
Thank you for the review :)